tests/networking-hosts.nix

```
@@ -0,0 +1,20 @@
+{ config, pkgs, ... }:
+
+{
+  networking.hosts = {
+    "127.0.0.1" = [ "my.super.host" ];
+    "10.0.0.1" = [ "my.super.host" "my.other.host" ];
+  };
+
+  test = ''
+    set -v
+    echo checking /etc/hosts file >&2
+
+    file=${config.out}/etc/hosts
+
+    grep '127.0.0.1' $file | head -n1 | grep localhost$ || exit 1
```

```suggestion
    grep '127.0.0.1' $file | head -n1 | grep localhost$
```

Can you remove the `|| exit` clauses?

---

Are you sure? I was under the impression the goal was to know which statement was failing because of the numbered exit codes.

---

My preference would be to not have different exit codes on failure